doc/user/content/sql/types/boolean.md

```diff
@@ -30,14 +30,15 @@ Detail | Info
 
 You can [cast](../../functions/cast) `boolean` to:
 
-- [`text`](../text)
 - [`int`](../int)
+- [`text`](../text)
```

Comment:
Sorry that this wasn't clear in my last review, but the information I think needs to be propagated to the individual types' pages is the cast context.